effekt/jvm/src/test/scala/effekt/core/VMTests.scala

```diff
@@ -695,8 +695,8 @@ class VMTests extends munit.FunSuite {
       dynamicDispatches = 783,
       patternMatches = 9464,
       branches = 14696,
-      pushedFrames = 7598,
-      poppedFrames = 7574,
+      pushedFrames = 7620,
+      poppedFrames = 7596,
```

Comment:
VM tests now show that a few more frames were pushed&popped (I guess because of `string::toInt`-related changes). cc @b-studios

Reply:
The newest change rolls this back a little:
<img width="240" alt="Screenshot 2025-05-13 at 20 34 28" src="https://github.com/user-attachments/assets/406511ef-00d8-4f49-bfd1-e28a091edad5" /><img width="242" alt="Screenshot 2025-05-13 at 20 34 48" src="https://github.com/user-attachments/assets/a060c391-8a90-42e6-910d-4d52787610d3" />